Classes/EventStore/Normalizer/ValueObjectNormalizer.php

```diff
@@ -52,6 +54,25 @@ public function supportsDenormalization($data, $className, $format = null): bool
         }
     }
 
+    public function normalize($object, string $format = null, array $context = [])
+    {
+        if (!$object instanceof \JsonSerializable && (!$object instanceof \Stringable || !method_exists($object, '__toString'))) {
```

Comment:
Objects with a `__toString` method automagically implement the `Stringable` interface. I don't think that we have to check both

Reply:
We do - trust me, I tried ;)

Reply:
can you elaborate what you tried? I'm curious :)

Reply:
I believe that I created a class with the `__toString` method, but no `implements \Stringable` and it wasn't caught - while writing I'm in doubt whether it was another condition that triggered my error 🤔

Reply:
ah OK. According to the [docs](https://www.php.net/manual/en/class.stringable.php)...
> Unlike most interfaces, Stringable is implicitly present on any class that has the magic [__toString()](https://www.php.net/manual/en/language.oop5.magic.php#object.tostring) method defined